scripts/workers/submission_worker.py

```diff
@@ -446,20 +446,23 @@ def main():
     sys.path.append(COMPUTE_DIRECTORY_PATH)
 
     load_active_challenges()
-
     connection = pika.BlockingConnection(pika.ConnectionParameters(
-        host='localhost'))
+        host=RABBITMQ_PARAMETERS.get('HOST', None)))
```

Comment:
We cannot use None as host, So `HOST` key must exist in `RABBITMQ_PARAMETERS`.

Also you can safely remove `None` everywhere.